packages/mui-base/src/Dropdown/Dropdown.test.tsx

```diff
@@ -125,6 +125,104 @@ describe('<Dropdown />', () => {
     expect(menuItems[1]).toHaveFocus();
   });
 
+  it('should focus first item when down arrow key opens the menu', () => {
```

Comment:
These tests belong to the Menu as the dropdown is more generic and will support more than just a menu in the future. Please move it to Menu.test.tsx and adapt it to look similar to tests there.